doc/classes/VehicleWheel3D.xml

```diff
@@ -18,6 +18,20 @@
 				Returns [code]null[/code] if the wheel is not in contact with a surface, or the contact body is not a [PhysicsBody3D].
 			</description>
 		</method>
+		<method name="get_contact_normal" qualifiers="const">
+			<return type="Vector3" />
+			<description>
+				Returns the normal of the suspension's collision in world space if the wheel is in contact.
+				Otherwise, returns a vector pointing directly along the suspension axis toward the vehicle, in world space.
```

Comment:
Merge into a single paragraph (for more compact description), and tweak the description to be more explicit:

```suggestion
				Returns the normal of the suspension's collision in world space if the wheel is in contact. If the wheel isn't in contact with anything, returns a vector pointing directly along the suspension axis toward the vehicle in world space.
```